ax/models/torch/botorch_modular/sebo.py

```diff
@@ -233,16 +232,34 @@ def optimize(
             candidates, a tensor with the associated acquisition values, and a tensor
             with the weight for each candidate.
         """
+        # Prepare arguments for optimizer
+        optimizer_options_with_defaults = optimizer_argparse(
+            self.acqf,
+            optimizer_options=optimizer_options,
+            optimizer="optimize_acqf_homotopy",
+        )
+
+        _tensorize = partial(torch.tensor, dtype=self.dtype, device=self.device)
+        ssd = search_space_digest
+        bounds = _tensorize(ssd.bounds).t()
```

Comment:
This doesn't seem to add any value
```suggestion
        bounds = _tensorize(search_space_digest.bounds).t()
```